### ISSUE TITLE
#a11y Label description for wp_auto_update_core

### ISSUE THREAD

**Issue by joedolson** [obenland/shiny-updates#69]
The description field for checkbox fields in settings should be explicitly associated with the input field. To do that, add `aria-labelledby="<?php echo esc_attr( $args['label_for'] ); ?> <?php echo esc_attr( $args['label_for'] . '_description' ); ?>"` to the input field and `id="</php echo esc_attr( $args['label_for'] . '_description' ); ?>"` to the description paragraph. This will cause both fields to be used as labels for the control.

If descriptions are added to other fields, they'll need the same treatment.
